seminar3/seminar3-netflix/src/api/movie.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from "express";
+import { Movie } from '../interface/Movie';
+import { data } from "../database/movie";
+
+const getMovie = (req:Request, res:Response) => {
+    //req.param 찍어보기
+    console.log(req.params.mediaId);
+    //console.log(typeof(req.params.mediaId)); //string
+
+    //클라이언트가 요청하는 mediaId의 영화 정보(json) 넘겨주기
+    const movie = data.find((m: Movie) => {
+        return m.title === req.params.mediaId;
+    });
+
+    if (movie) {
+        return res.status(200).json(movie);
+    } else {
+        return res.status(400).json({errorMessage:" Movie was not found... "});
+    }
```

Comment:
if-else로 예외를 핸들링하는 것보다, 예외처리가 필요한 부분만 if를 걸어서 breaking 하는 것을 더 추천드립니다.
모든 예외 처리를 통과한 경우 제일 하단에 있는 return을 통해 성공적으로 로직을 끝내는 것이 더 좋아보여요!

```suggestion
    if (!movie) {
        return res.status(400).json({errorMessage:" Movie was not found... "});
    } 
    
    return res.status(200).json(movie);
```

Reply:
우어 !! 네네 훨씬 코드가 깔끔해지는 것 같아요 ! 좋은 코드 리뷰 감사합니다 😄
덕분에 좋은 예외 처리 구조 배울 수 있었어요 :) 앞으로 신경 쓰겠습니당 !